carve.lic

```diff
@@ -0,0 +1,118 @@
+=begin
+  Suggestions and contributions are welcome: https://github.com/rpherbig/dr-scripts
+=end
+
+custom_require(%w(common common-crafting))
+
+class Carve
+  include DRC
+  include DRCC
+
+  def initialize(args:[])
+    settings = get_settings
+    @bag = settings.crafting_container('backpack')
+    @finish, @type, @chapter, @recipe_name, @material, @noun = args
+
+    case @type
+    when /carving/
```

Comment:
To me this indicates that we should roll this into `shape` (and maybe rename it to `engineer`?). Don't worry about it as part of this PR, but we'll want to look into it later.